### PR TITLE
invitation images, invitation templates and invite respondents

### PR DESCRIPTION
**Pull request by DfernandezNipo**
[AB#104031](https://dev.azure.com/niposoftware/15ce0e91-931d-4fbf-9169-8c3dde412b54/_workitems/edit/104031)

### CODE REVIEW

File: Postman/SurveyInvitationTemplates.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
+{
+	"info": {
+		"_postman_id": "411031ca-e7aa-4aa6-ac31-3391cbc3117a",
+		"name": "SurveyInvitationTemplates",
+		"description": "A controller for manage invitation templates.",
```
Comment (MartinRijks) [inline Postman/SurveyInvitationTemplates.postman_collection.json:5]:
```suggestion
		"description": "A controller for managing invitation templates.",
```

File: Postman/SurveyInvitationTemplates.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"{{SurveyId}}",
+						"InvitationTemplates",
+						"{{InvitationTemplateId}}"
+					]
+				},
+				"description": "This method updates the ODIN script of a specific survey."
```
Comment (MartinRijks) [inline Postman/SurveyInvitationTemplates.postman_collection.json:202]:
```suggestion
				"description": "This method updates an invitation template."
```

File: Postman/SurveyInviteRespondents.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+						"{{origin}}"
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"InviteRespondents",
+						"SurveysInvitationStatus"
+					]
+				},
+				"description": "This method gets invitation status."
```
Comment (MartinRijks) [inline Postman/SurveyInviteRespondents.postman_collection.json:133]:
```suggestion
				"description": "This method gets an invitation status."
```

File: Postman/SurveyInviteRespondents.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"{{SurveyId}}",
+						"InviteRespondents",
+						"SurveyBatchesStatus"
+					]
+				},
+				"description": "This method gets invitation batches status."
```
Comment (MartinRijks) [inline Postman/SurveyInviteRespondents.postman_collection.json:214]:
```suggestion
				"description": "This method gets an invitation batch status."
```

File: Postman/SurveyInviteRespondents.postman_collection.json [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+			},
+			"response": []
+		},
+		{
+			"name": "Get Invitation batch status",
```
Comment (MartinRijks) [inline Postman/SurveyInviteRespondents.postman_collection.json:219]: This one has (almost) the same name as the previous one, maybe the difference should be indicated in the name.

File: Postman/SurveyInviteRespondents.postman_collection.json (hunk 4 above)
Comment (DfernandezNipo) [reply Postman/SurveyInviteRespondents.postman_collection.json:214]: This one get status of all batches

Comment (MartinRijks) [reply Postman/SurveyInviteRespondents.postman_collection.json:214]: OK, the I updated my comment.

**Review by MartinRijks — APPROVED** [review]

**Review by AlbertoBelver — APPROVED** [review]

**Pull request merged**